CefSharp/CefSharpSettings.cs

```diff
@@ -20,8 +20,30 @@ static CefSharpSettings()
             LegacyJavascriptBindingEnabled = false;
             WcfTimeout = TimeSpan.FromSeconds(2);
             SubprocessExitIfParentProcessClosed = true;
+            JavascriptBindingPropertyName = "CefSharp";
+            JavascriptBindingPropertyNameCamelCase = "cefSharp";
         }
 
+        /// <summary>
+        /// Configure the browser binding property name.
+        /// Use a custom property name when calling registred objects, example 'CefSharp.BindObjectAsync'.
+        /// Alphanumeric and underscores characters only.
+        /// The default value is 'CefSharp'
+        ///
+        /// NOTE: There are two properties to be mindful of, this and <see cref="JavascriptBindingPropertyNameCamelCase"/>
+        /// </summary>
+        public static string JavascriptBindingPropertyName { get; set; }
```

Comment:
- We we need two properties for this? Camel case can be derived from the full name
- What happens if both properties are exactly the same?

If they both end up being the same name we should only generate one set of objects.

Reply:
How about exposing a single property and then manipulate the string to pascal/camel case on the first char of the property before injecting that way you can keep both objects and cater for either preference?

For example:
https://stackoverflow.com/a/36579722

Or simply leave it down to preference completely, and only have a single object.

Reply:
Single property sounds fine, if the first character is already lower we should only create one set of objects. 

https://github.com/cefsharp/CefSharp/blob/master/CefSharp/Internals/JavascriptObjectRepository.cs#L543 should be refactored if we add an additional method for camel case conversion.